src/manuscript.tex

```diff
@@ -41,11 +41,12 @@
 		% Author declaration
 		\makedeclaration
 
-		% Acknowledgements
-		\input{acknowledgements}
-
 		% Preface
 		\input{preface}
+		
+        % Acknowledgements

```

Comment:
There's a bunch of spaces vs. tabs issues like this across many of the commits.  Can you please address them? :-)